sim/druid/typhoon.go

```diff
@@ -39,7 +43,7 @@ func (druid *Druid) registerTyphoonSpell() {
 
 		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
 			spell.WaitTravelTime(sim, func(sim *core.Simulation) {
-				baseDamage := 1190 + 0.193*spell.SpellPower()
+				baseDamage := 1323 + (spell.SpellPower() * spellCoeff)
```

Comment:
remove manual spell power calculation and add spellCoeff to the spells BonusCoefficient value

Reply:
done using CalcScalingSpellAverageEffect() and also moved out GaleWinds and implemented it in talents